Makefile.js

```diff
@@ -705,7 +723,16 @@ target.gensite = function(prereleaseVersion) {
                 } else {
                     title = "Documentation";
                 }
-                text = `---\ntitle: ${title}\nlayout: doc\n---\n<!-- Note: No pull requests accepted for this file. See README.md in the root directory for details. -->\n\n${text}`;
+                text = [
+                    "---",
+                    `title: ${title}`,
+                    "layout: doc",
+                    `edit_link: https://github.com/eslint/eslint/edit/master/${baseName}`,
+                    "---",
+                    "<!-- Note: No pull requests accepted for this file. See README.md in the root directory for details. -->",
+                    "",
+                    text
+                ].filter(x => x).join("\n");
```

Comment:
Why is `.filter(x => x)` used here? It seems like this will result in the `""` on line 733 getting removed, which seems redundant.